components/prism-birb.js

```diff
@@ -0,0 +1,23 @@
+Prism.languages.birb = Prism.languages.extend('clike', {
+	'class-name': [
+		/\b[A-Z](?:[\d_]*[a-zA-Z]\w*)?\b/,
+
+		// matches variable and function return types (parameters as well).
+		/\b[A-Z]\w*(?=\s+\w+\s*[;,=()])/
+	],
+	'string': {
+		pattern: /r?("|')(?:\\.|(?!\1)[^\\])*\1/,
+		greedy: true
+	},
+	'keyword': /\b(?:assert|break|case|class|const|default|else|enum|final|follows|for|grab|if|nest|next|new|noSeeb|return|static|switch|throw|var|void|while)\b/,
+	'variable': /\b[a-z_]\??\w*\b/,
```

Comment:
What is the `\??` for?

Also, please move it after `operator` since `operator` is also a token of C-like.

Reply:
it is not an operator, this is to match smth like `int?`, datatypes ending in `?` are nullable. Though, I have misplaced it

Reply:
I've removed it as we are still testing/implementing it and haven't included it in the docs yet